packages/gatsby-plugin-sharp/src/index.js

```diff
@@ -137,13 +139,60 @@ function queueImageResizing({ file, args = {}, reporter }) {
   }
 }
 
+// Get blurhashed and transform to base64
+async function fileBlurhashedToBase64(
+  absolutePath,
+  width,
+  height,
+  componentX,
+  componentY
+) {
+  // Load image
+  const image = await loadImage(absolutePath, width, height)
```

Comment:
would this be faster if we pre-processed the image to a smaller one e.g. 400px wide? Some images in people's site can be quite large. Not sure how well node-canvas scales